### ISSUE TITLE
Enable more Stripe payment options

### ISSUE THREAD

**Issue by zegl** [polarsource/polar#1604]
Enable more Stripe payment options in the checkout flow.

I don't see any reason to _not_ activate all of the "bank redirects" and "real-time payments" payment methods that Stripe support (Bancontact, BLIK, iDEAL, etc).

**Comment by JaffaKetchup** [comment]
This would be great, as many of them have lower fees as well!

**Comment by birkjernstrom** [comment]
Truly sorry for the delay here @JaffaKetchup - I need to thoroughly review my GitHub notification settings & workflow. 

I've enabled a lot of new payment methods today. Beyond standard credit cards & wallets (Cash App, Apple Pay, Google Pay etc) we now support:
- Cartes Bancaires (France - pending)
- Bank redirects: Bancontact (Belgium), EPS (Austria), giropay (Germany), iDEAL (Netherlands), Przelewy24 (Poland)
- BNPL: Affirm, Afterpay / Clearpay, Klarna

We could also add support for *Bank debits*, e.g SEPA Direct Debit. However, they do not offer instant confirmation which would impose some nuances we would need to address in our product/flows. I've therefore created a dedicated issue for those & to gauge interest.
https://github.com/polarsource/polar/issues/2390

**Comment by birkjernstrom** [comment]
One more important note for prosperity here: Not all of these payment methods offer recurring billing or are available in all countries of course. However, we now have as wide support we can at this stage in terms of our Stripe integration and that we can offer as many of them as possible at checkout depending on whether it's issue funding or subscriptions.

**Comment by JaffaKetchup** [comment]
No, no worries at all. We haven't actually used Polar yet, turns out we haven't required it yet.

It would be awesome if some of the increased revenue from lower fees from the local payment methods could be passed onto the users - for example, in the UK, all iDEAL payments only cost 25p. It would be great if this meant we saw lower costs when payments are made through these methods as well. See https://stripe.com/gb/pricing/local-payment-methods for local methods in the UK.

**Comment by birkjernstrom** [comment]
Got it @JaffaKetchup - let me know if you ever want to chat about getting setup.

Just to make sure I understand. Your feature request is to be able to customize whether the end-user should pay for the Stripe fees? That's how I interpret the first sentence, but the second part made me unsure. To clarify in terms of our pricing: Polar charges 5% + PSP fees. So the latter would reflect the costs per payment method.

**Comment by JaffaKetchup** [comment]
> To clarify in terms of our pricing: Polar charges 5% + PSP fees. So the latter would reflect the costs per payment method.

Ah, I thought I had read something about 10% inc. payment processor fees, but maybe that was another service. This is what I was asking for. Thanks for clearing it up!

> Your feature request is to be able to customize whether the end-user should pay for the Stripe fees?

That's an intresting idea as well though, and I would like something like this! Maybe a third option (in addition to end-users cover and don't cover fees) could be for it to be optional on the backer's end to cover (a) the Polar fees (b) the PSP fees (a little more difficult to estimate perhaps)?

**Comment by birkjernstrom** [comment]
> Ah, I thought I had read something about 10% inc. payment processor fees, but maybe that was another service. This is what I was asking for. Thanks for clearing it up!

No problem and of course. We changed our pricing on January 9th to 5% from 10%. Our pricing has always been excl. PSP fees, but we've covered them and continue to do so until end of March this year. So completely understandable that it's a bit confusing, but all in a positive way :-) So to clarify ahead our current pricing is: 5% + PSP fees (after March 31st, 2024 – covered until then).

> That's an intresting idea as well though, and I would like something like this! Maybe a third option (in addition to end-users cover and don't cover fees) could be for it to be optional on the backer's end to cover (a) the Polar fees (b) the PSP fees (a little more difficult to estimate perhaps)?

Thanks for the great feedback – I'm glad I asked to clarify. Unfortunately, it's a bit tricky how we would offer this cohesively now with subscriptions too. But it's certainly not impossible and something I've seen in the wild that others do here and there. Our main focus now though is in helping developers get more subscribers and funding. So in transparency, I don't think we'll build this in the short-term until we get to the stage of optimizing what I hope is a successful pipeline of subscribers & funding to developers – I hope that makes sense? :)
